src/CONST.ts

```diff
@@ -6924,6 +6924,10 @@ const CONST = {
     ILLUSTRATION_ASPECT_RATIO: 39 / 22,
 
     OFFLINE_INDICATOR_HEIGHT: 25,
+
+    BILLING: {
+        TYPE_FAILED_2018: 'typeFailed2018',
```

Comment:
While QAing this on staging I realized this got changed at the last minute to be `typeFailed2018`, when the correct value is `failed_2018`. I'll open a PR to fix it

Reply:
https://github.com/Expensify/App/pull/59762